content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,546 @@
+---
+title: Running Kubelet in Standalone Mode
+content_type: tutorial
+weight: 10
+---
+
+<!-- overview -->
+
+This tutorial shows you how to run a standalone Kubelet instance.
+
+In standalone mode, the Kubelet allows you to manage containers using PodTemplates,
+which brings the benefits of running tightly coupled applications as a single unit while
+leveraging a subset of advanced features, like CNI networking, without the need
+for the entire Kubernetes feature set.
+
+## {{% heading "objectives" %}}
+
+* Install and configure `containerd`, `cni-plugins`, and `kubelet` on a Linux system.
+* Run `containerd` and `kubelet` as `systemd` services.
+* Launch a Pod running `nginx` that listens to requests on TCP port 80 on the Pod's IP address.
+
+## {{% heading "prerequisites" %}}
+
+- Admin (root) access to a Linux system running `systemd`.
+- Access to the Internet to download the various components needed for the tutorial, such as:
+    - A {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+that implements the Kubernetes {{< glossary_tooltip term_id="cri" text="(CRI)">}}.
+    - Network plugins {{< glossary_tooltip text="Container Networking Interface (CNI)" term_id="cni" >}}
+- Required CLI tools: `curl`, `tar`.
+- Suggested CLI tools: `jq`.
+
+<!-- lessoncontent -->
+
+## Download, install, and configure the components
+
+{{% thirdparty-content %}}
+
+{{< note >}}
+Download the latest available versions of the required packages.
+The relative path to the downloaded software is assumed to be the `sources` directory.
+{{< /note >}}
+
+### Create the sources directory
+
+```shell
+mkdir sources
+cd sources/
+```
+
+### Swap configuration
+
+The default behavior of kubelet is to fail to start if swap memory is detected on a node.
+This means that swap should either be disabled or tolerated by kubelet.
+
+{{< note >}}
+If you configure the kubelet to tolerate swap, the kubelet still configures Pods (and the
+containers in those Pods) not to use swap space. To find out how Pods can actually
+use the available swap, you can read more about
+[swap memory management](/docs/concepts/architecture/nodes/#swap-memory) on Linux nodes.
+{{< /note >}}
+
+If you have swap memory enabled, either disable it or add `failSwapOn: false` to the
+kubelet configuration file.
+
+To disable swap temporarily:
+
+```shell
+sudo swapoff -a
+```
+
+To make this change persistent across reboots:
+
```

Comment:
I'm not at all keen on this. It's fine to run this if you know what it does, but to a learner they might lose their swap configuration and then not know how to get it back.

Better to advise them to edit `/etc/fstab` rather than offering them a potentially confusing one liner.